src/openrct2-ui/drawing/engines/opengl/TextureCache.cpp

```diff
@@ -201,7 +201,7 @@ void TextureCache::CreateTextures()
 
 void TextureCache::GeneratePaletteTexture()
 {
-    static_assert(PALETTE_TO_G1_OFFSET_COUNT + 5 < 256, "Height of palette too large!");
+    static_assert(PALETTE_TOTAL_OFFSETS + 5 < 256, "Height of palette too large!");
```

Comment:
Can you document why the +5 i.e. name the 5 to a constant and also 256.

Reply:
This is something that existed before - All the palette maps have a width of 256px. Most of them have a height of 1px, but there's a single one that has a height of 5px (1280 total colors). It's palette map 32, and it's related to water colors. When assembling the texture page for the palette, the max is 256x256px, so this assert accounts for that. 

The only change I made here is updating the old palette map constant (144) to the new one (192).